templates/intro-workspaces/README.ipynb

```diff
@@ -8,17 +8,17 @@
     "\n",
     "**⏱️ Time to complete**: 10 min\n",
     "\n",
-    "Welcome! You are currently in a Workspace, which is a persistent cloud IDE connected to a Ray cluster.\n",
+    "Welcome! You are currently in a Workspace, which is a persistent cloud IDE connected to a Ray Cluster.\n",
```

Comment:
This should be lowercase, per https://docs.ray.io/en/latest/cluster/getting-started.html